client/src/hooks/datafiles/mutations/toolbarAppUtils.ts

```diff
@@ -47,7 +44,7 @@ export function getAllocationForToolbarAction(
     return (
       getAvailableHPCAlloc(
         allocationsState.active,
-        appObj.definition.jobAttributes.execSystemId
+        appObj.execSystems[0].host
```

Comment:
The `getAllocationForToolbarAction` is still returning the `allocationsState.portal_alloc` by default, even if that alloc is not active, or is not valid for the compress/extract exec system

Reply:
just pushed up changes to consider this, thanks for this note!